tc/core/tc2halide.cc

```diff
@@ -492,6 +495,8 @@ Expr reductionUpdate(Expr e) {
   return Call::make(e.type(), kReductionUpdate, {e}, Call::Intrinsic);
 }
 
+// translate a single TC comprehension/statement to Halide component.
+// funcs, bounds, reductions will be populated
```

Comment:
Since you are documenting this function, you might as well explain what
those are and how they relate to the "Halide component".